.github/ISSUE_TEMPLATE/general-issue-template.md

```diff
@@ -4,11 +4,12 @@ about: Template for any fqm-execution issue reports
 title: ''
 labels: ''
 assignees: ''
-
 ---
 
 ### Summary
 
+#### External Tracking Ticket
```

Comment:
```suggestion
### External Tracking Ticket
```
Thoughts?

Reply:
Or was the smaller header on purpose...if so ignore

Reply:
Smaller header on purpose. But I guess it can be up for discussion.